Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/SANSILLAutoProcess.py

```diff
@@ -533,6 +544,11 @@ def processSample(self, i, flux_name, sample_transmission_name, beam_name,
                 )
 
         panel_outputs = self.getPropertyValue('PanelOutputWorkspaces')
+        n_wedges = self.getProperty('NumberOfWedges').value
```

Comment:
You are getting this and the output type several times, can you not create a local or class member variable and get once?

Reply:
I created a class member variable.